README.adoc

```diff
@@ -418,3 +420,22 @@ In response you should get ID of created item, using that you can retrieve the w
 ----
 $ ./bin/wit-cli show workitem --id <ID> -H localhost:8080 --pp
 ----
+
+== Prod preview setup
+
+In order to setup wit to work against link:https://prod-preview.openshift.io/[prod preview] one needs to update `token.publickey` in config.yaml.
+
+NOTE: Public key can be obtained from https://sso.prod-preview.openshift.io/auth/realms/fabric8/
+
+Also, the following environment variables must be set:
+
+* `export F8_DEVELOPER_MODE_ENABLED=1`
+* `export F8_OPENSHIFT_TENANT_MASTERURL="https://console.free-int.openshift.com/"`
+* `export F8_KEYCLOAK_REALM="fabric8"`
```

Comment:
>  fabric8-test most likely does not have an account setup with OSO tokens for free-int etc

@ibuziuk This might be true... so better not to induce users to get failures. Let's keep your setup.